api/routes/application/routes.py

```diff
@@ -78,9 +79,10 @@ def post(self):
         add_new_forms(forms=empty_forms, application_id=application.id)
         return application.as_dict(), 201
 
-    def get_by_id(self, application_id):
+    def get_by_id(self, application_id, **kwargs):
```

Comment:
Is there a need to capture and pass all kwargs? It looks like we only need `with_questions_file`

Reply:
Fixed